src/main/thrift/parquet.thrift

```diff
@@ -529,7 +596,15 @@ struct DataPageHeader {
   /** Encoding used for repetition levels **/
   4: required Encoding repetition_level_encoding;
 
-  /** Optional statistics for the data in this page**/
+  /**
+   *  Optional statistics for the data in this page.
+   *
+   * For filter use-cases populating data in the page index is generally a superior
+   * solution because it allows readers to avoid IO, however not all readers make use
+   * of the page index.  For best compatibility both should be populated. If the writer
```

Comment:
This appears to contradict the docs for page index

> Readers that support ColumnIndex should not also use page statistics. The only reason to write page-level statistics when writing ColumnIndex structs is to support older readers (not recommended).

Reply:
Agree, this seems to switch the suggestion from not writing both to writing both. Are we sure this PR warrants a 180 degree turn on this?

Reply:
Good catch. I don't think we need to populate both here. Perhaps we should rephrase it to be consistent. @emkornfield

Reply:
I think this has likely become tangential to the PR I'm going to revert this.